docs/server/source/server-reference/configuration.md

```diff
@@ -248,6 +251,38 @@ export BIGCHAINDB_WSSERVER_PORT=9985
 }
 ```
 
+## wsserver.advertised.scheme, wsserver.advertised.host and wsserver.advertised.port
+
+These settings are for the advertising the Websocket URL to external clients in
+the root api endpoint. These configurations might be useful if your deployment
```

Comment:
Please write `API` in all-capital letters when writing it in prose, because it's an acronym.